\documentclass[main]{subfiles}

ABSTRACT. 


DNA G-quadruplex (G4) small molecule ligands such as CX-5461 have been proposed as therapeutic agents in genomically unstable cancers with defects in DNA repair pathways such as homologous recombination (HR). Although CX-5461 is currently being tested in a clinical trial (NCT02719977) for BRCA1/2-mutated cancers, the spectrum of defects in genome maintenance pathways that render cancer cells vulnerable to G4 ligands is not known. To address this question, we conducted a CRISPR-Cas9 genetic screen in HCT116 cell line using two G4 ligands, CX-5461 and pyridostatin (PDS), to examine lethal gene-drug interactions with 480 genes including genome maintenance genes. We re-identified several members of the HR and Fanconi anemia (FA) pathways, as mechanistically predicted, indicating the sensitivity of the primary screen. We also identified and validated two members of the DNA damage response (DDR)-associated ubiquitin signaling, UBE2N and RNF168, that have previously not been associated with G4 stabilization-associated DNA damage. We show that (1) targeting of RNF168 and UBE2N exhibits significantly lower cell survival in the presence of CX-5461 compared with the vehicle or an unrelated compound that binds GC rich DNA but not G4 sequences, (2) nuclear ubiquitination response increases with CX-5461 treatment and co-localizes with G4 regions, and (3) UBE2N inhibition acts synergistically with G4 stabilization by CX-5461. In conclusion, we have uncovered a novel mechanism of G4 structure resolution that involves chromatin ubiquitin modification through RNF168 and UBE2N. Additionally, our results  potentially expand the pool of patients who could benefit from G4 stabilizers.

%Abstract by Tehmina
%formation at guanine-rich sequences can interfere with the normal progression of many cellular processes, including DNA replication, repair, and recombination. A number of factors involved in DNA repair are also required for the resolution of these structures. Deficiency of any of these factors results in genome instability at these sites. Many G4 ligands have been reported that promote genome instability at G4 sites in the absence of any of these factors. One such ligand, CX-5461, has been shown to increase cytotoxicity in BRCA1/2 deficient cancer cells and is currently being tested in phase I/II clinical trial for breast malignancies. To gain a comprehensive understanding of how G4 structures are resolved and to broaden the spectrum of tumors that could be targeted with G4 ligands, we conducted a pooled subgenomic CRISPR-Cas9 genetic screen in human colorectal cancer cell line HCT116. We used two G4 ligands, CX-5461 and pyridostatin (PDS), to examine gene-drug interactions with 480 genes including genome maintenance genes. Overall, there were many overlaps between the top hits from CX-5461 and PDS screens.  The list of top hits included genes previously known to resolve G4 structures. We also identified and validated genes and pathways that are not known to be involved in G4 structure resolution, including DNA damage response (DDR)-associated ubiquitin signaling factors, UBE2N and RNF168. We further characterized the mechanism of regulatory ubiquitin signaling in G4 structure resolution using cell-based assays. In conclusion, we have uncovered a novel mechanism of G4 structure resolution that involves chromatin ubiquitin modification.  These results also suggest that patients with UBE2N- and RNF168-mutated tumors could potentially benefit from CX-5461 therapy. In addition, combination therapy using inhibitors of DDR-associated ubiquitination could be used to enhance the effectiveness of CX-5461 in many cancers.

%Abstract by Sam
%DNA G-quadruplex (G4) small molecule ligands such as CX-5461 have been proposed as therapeutic agents in genomically unstable cancers with DNA repair deficiencies. Although homologous recombination repair deficiency (HRD) has been identified as a mechanism of synthetic lethality currently being tested in a clinical trial(NCT02719977), the spectrum of DNA repair and genome maintenance deficiencies that may be synthetic lethal to G4 binding molecules is not known. To address this question, we conducted a CRISPR-Cas9 genetic screen using two G4 ligands, CX-5461 and pyridostatin (PDS), to examine gene-drug interactions with 480 genes including genome maintenance genes. We re-identified several members of the HRD and FA pathways, as mechanistically predicted, indicating the sensitivity of the primary screen. We also identified and validated two members of the DNA damage response (DDR)-associated ubiquitin signaling factor mechanism, UBE2N and RNF168, that have not been associated with G4 stabilization mediated DNA-damage. We show that (i) knockdown of RNF168 and UBE2N exhibits increased loss of fitness in the presence of CX-5461 compared with vehicle or an unrelated compound that binds GC rich DNA but not G4 sequences (ii) ubiquitination signal increases with CX-5461 treatment and co-localizes with  DNA damage loci that are enriched at G4   (iii) Synergism between UBE2N inhibition and G4 binding compound - CX-5461 occurs in cytotoxicity assays. In conclusion, we have uncovered a novel mechanism of G4 structure resolution that involves chromatin ubiquitin modification  through RNF168 and UBE2N. Our results potentially expand the pool of patients who could benefit from G4 stabilizers.
