%%\documentclass[main]{subfiles}
\section*{Discussion}

We describe a subgenomic CRISPR-Cas9 dropout screen to identify multiple potentially lethal interactions between G4 drugs, CX-5461 and PDS, and genes involved in genome maintenance in Tp53-wildtype HCT116 cell line. To eliminate gene hits resulting from CX-5461-associated inhibition of RNA POL I complex, parallel drug screens with an RNA POL I inhibitor, BMH-21, were conducted. Among the top candidate genes, we identified genes known to resolve G4 structures (BRCA2, REV1) or described previously in G4-drugs screens in \textit{C. elegans} (ATM, POLQ), and prioritized UBE2N and RNF168 genes. The mechanism underlying selective elimination of tumor cells with compromised UBE2N- and RNF168-mediated ubiquitination, upon exposure to G4 ligands, was further characterized.

Altogether, we used four different G4 drug treatments, CX-5461 and PDS, each at IC$_{50}$ and IC$_{30}$ doses for our screens. Using this strategy, we identified 23 genes whose sgRNA targeting increased the tumor cells sensitivity in at least two G4 drug treatment groups, and not in BMH-21-treatment group. The sensitivity of our screening approach was demonstrated by the re-identification of lethal G4 drug interaction with BRCA2 and REV1\cite{Eddy2014,Zimmer2015,Xu2017} as well as ATM and ATR\cite{Negi2015a,Quin2016}. In addition, screen follow-up with competitive growth assays validated, to varying extent, four genes of interest with UBE2N and RNF168 exhibiting the strongest lethal interactions. In order to extend our findings beyond one cell line and/or sgRNA-mediated gene knockout, we demonstrated increased sensitivity to G4 drugs in (1) HCT116 UBE2N-null cells generated elsewhere\cite{Thorslund2015}, as compared to WT cells, (2) MDA-MB-231 breast cancer cells after sgRNA-mediated targeting of UBE2N, and (3) HCT116 cells after pharmacological inhibition of UBE2N inhibitor, as compared to either drug alone. We also attempted to knockdown or knockout RNF168 and RNF8 gene expression in multiple cell lines albeit without any success.  
 
The importance of UBE2N-, RNF8- and RNF168-mediated ubiquitin signaling and posttranslational histone ubiquitination in DNA DSB repair is well appreciated.  UBE2N, an E2 ubiquitin-conjugating enzyme, and RNF8 and RNF168, E3 ubiquitin ligases, are recruited to DNA DSBs\cite{Kolas2007,Mailand2007,Huen2008,Stewart2009}. 
%[insert appropriately Hofmann JBC2001]. 
They trigger a non-proteolytic ubiquitination cascade which contributes to DNA repair and repair pathway choice by recruiting additional factors to the DSB sites. UBE2N acts together with one of its two non-catalytic variants, UBE2V1 or UBE2V2, previously known as UEV1 and MMS2, respectively\cite{Hofmann1999a,Hodge2016c}. However, UBE2V1 only participates in cytoplasmic ubiquitination. UBE2N-UBE2V2 heterodimer binds RNF8, another E3 ubiquitin ligase. This interaction is required for the formation of K63-linked ubiquitin chains on H1-type linker histones which provides a binding platform for RNF168 via UDM1 domain\cite{Hofmann2001,Eddins2006,Kolas2007,Campbell2012,Thorslund2015}. RNF168 then catalyzes the ubiquitination of H2A-type histones at K13/K15, leading to recruitment of downstream DNA repair factor, 53BP1, which promotes non-homologous end-joining (NHEJ)\cite{Gatti2012a,Mattiroli2012,Fradet-Turcotte2013a}. HR is also suppressed by RIF1 in the G1 phase of the cell cycle. However, during S/G2 phase, RAP80 recruitment to the extended K63-linked ubiquitin chains and loss of RIF1-dependent suppression leads to recruitment of BRCA1 complex and ultimately homologous recombination (HR)\cite{Mailand2007,Gatti2012a,Mattiroli2012}. 

The contribution of regulatory chromatin ubiquitination to G4 structure resolution and to the repair of G4-associated DNA damage has not been investigated. To explore the underlying mechanism of gene-drug interactions, we examined activation of nuclear ubiquitin signaling and subsequent chromatin ubiquitination upon exposure to G4 drugs. We showed that CX-5461 treatment resulted in significant induction of RNF168 foci in nuclei. Additionally, we observed the accumulation of conjugated ubiquitin (FK2) at DNA DSBs and of K63-linked ubiquitin chains within chromatin after CX-5461 treatment. Importantly, a significant number of CX-5461-induced FK2 foci overlapped with G4 sites marked by BG4 antibody, strongly suggesting that chromatin ubiquitination is induced not only in response to DNA DSBs but also in association with G4 structures stabilized by CX-5461. It would be interesting to further explore additional players of ubiquitin regulation during G4-associated DDR, and how ubiquitination cascade controls DNA repair pathway choice for DNA DSBs at G4 sites. 

RNF168 has been suggested to link H2A ubiquitination to PALB2- and RAD51-dependent HR by directly interacting with PALB2 and loading PALB2 onto damaged DNA which is redundant with BRCA1 function during HR\cite{zhang2009,Zhang2009a,sy2009,Luijsterburg2017a}. Interestingly, PALB2 was one of the hits in our screens although we did not explore its role in detail. RNF168 deficiency predisposes BRCA1 heterozygous mice to cancers, and these cells are  hypersensitive to PARP inhibitors\cite{zong2019}. Forced targeting of PALB2 to damaged chromatin bypasses RNF168 requirement for genome maintenance in BRCA1 haploinsufficiency. It would be interesting to determine if these tumors are also sensitive to CX-5461.  Previously, we showed that some BRCA1-mutated tumors that are resistant to PARP inhibitors and platinum salts are sensitive to  CX-5461\cite{Xu2017}. Inhibition of chromatin ubiquitin pathway may further increase the sensitivity of these tumors to CX-5461.
  
BLM recruitment to sites of replication fork stalling is also triggered by RNF8/RNF168-regulated ubiquitination\cite{Tikoo2013,Tripathi2018}. Recently, HERC2, a HECT E3 ligase, was shown to be critical for the retention of BLM and WRN on G4 DNA, and thus suppression of G4 DNA during the S-phase of the cell cycle\cite{Wu2018}. HERC2 also facilitates the assembly of UBE2N-RNF8 complex as well as maintains the levels of RNF168 at the DNA DSBs\cite{Bekker-Jensen2010}. Altogether, this suggests an important link between DDR-associated ubiquitin signaling and recruitment/assembly/retention of factors involved in DNA G4-structure resolution. Our sub-genomic library did not include HERC2 and functional interplay of these proteins requires further investigation. 

We also selected UBE2N and RNF168 for further characterization as UBE2N, RNF8 and RNF168 gene alterations are associated with multiple cancers. Previously, we examined the antiproliferative potency of CX-5461 and CX-3543 against a panel of breast cancer cell lines\cite{Xu2017}. Indeed, the MDA-MB-436 cell line with low gene-level copy number and reduced mRNA expression for both UBE2N and RNF168 genes, as determined from Cancer Cell Line Encyclopedia, exhibited the highest sensitivity to both CX-5461 and CX-3543 (Supplementary Figure~\ref{sfig:cell_line_sensitivity} and Supplemental Data)\cite{Barretina2012,Xu2017}. Although these might not be the only factors contributing to the observed sensitivity, it will still be interesting to further investigate the sensitivity of UBE2N/RNF8/RNF168-impaired tumors and cell lines to G4 drugs. 

Additional studies to validate other candidate genes identified in our screen, and to explore the underlying mechanisms of sensitivity to G4 drugs are also needed. Additional drug-gene interactions in tumor cells will be revealed by whole-genome screens, which could further broaden the spectrum of tumors sensitive to G4 drugs or inform combination therapeutic strategies.

In summary, we optimized and validated a sub-genomic CRISPR-Cas9 screen to determine genetic interactions with G4 drugs and demonstrated that cancer-associated UBE2N/RNF168 alterations or pharmacological suppression of DDR-associated ubiquitin signaling could be exploited in human cancers for therapy with CX-5461. In addition, our work has uncovered a novel role for regulatory ubiquitination pathway in DNA G4 structure resolution and damage repair.  

%% avoiding a discussion of contributions of NHEJ versus HR. However, if the reviewers comment on it, we could consider including the following discussion points.
% Loss of genes in NHEJ and HR pathways all increase sensitivity to CX-5461, suggesting the formation of DSBs  when cells are treated with CX-5461\cite{Xu2017}. 

%CX-5461 induces replication-associated DNA damage during S/G2 phase of the cell cycle. Our results demonstrate that within short time of CX-5461 treatment (1hr incubation), chromatin ubiquitination response mostly occurs in S phase and is involved in early processing of DNA damage induced by CX-5461. We propose a model in which breaks arise in S phase followed by induction of UBE2N-RNF8-mediated ubiquitin signaling and recruitment of RNF168 and PALB2 to the damage sites,  which might facilitate repair of the damage through HR. Alternatively, RNF168 and 53BP1 interaction might facilitate NHEJ. These two models are not mutually exclusive with HR being more dominant in S/G2 phase of the cell cycle, and NHEJ functions in all cell cycle phases. Although DNA damage and ubiquitination initially occur at S phase, during longer CX-5461 treatment, un-repaired damage might accumulate in G2 and also G1 phase of the next cell cycle. Further investigation is needed to characterize the detailed mechanism of how DSBs are formed and how HR and NHEJ cooperating in the repair of CX-5461 induced DNA damage.   

%%Advantages of our approach: 
%%Ability to observe phenotypes that could be missed in genome-wide screens esp with genome stability genes that would diminish early on. 

%%multiple drugs with overlapping functions

%%Speculation on how UBE2N-RNF8-RNF168-53BP1-NHEJ play a role. Our data support an earlier role (S/G2? phase) of these proteins in repairing replication-associated damage at DNA G4 sites. At least a fraction of DNA DSBs generated during S/G2 recruiting these factors (NHEJ dominant during G1 and late G2). Also likely, that damage is propagated into next cell cycle inducing further DSBs and a more potent phenotype. We, however, did not examine that

%%Alternative model, not mutually exclusive with the NHEJ model, suggests that RNF168-PALB2 axis promotes HR of the DSBs in S/G2. The choice between two pathways is determined by chromatin modification surrounding the DSBs.