%%\documentclass[main]{subfiles}
\section*{Introduction}

Folded G-quadruplex (G4) structures arise transiently at guanine-rich sequences in the DNA and RNA of multiple organisms, including microbes, yeast, plants, \textit{C. elegans}, mouse and human\cite{GELLERT1962,Sen1988,Schaffitzel2001,Verma2008,Hershman2008,Smargiasso2009,Beaume2013,Perrone2017,Saranathan2018}. G4 formation plays important roles in biological processes such as gene transcription, translation, telomere maintenance, DNA replication, and recombination\cite{Siddiqui-Jain2002,Morris2010,Lopes2011}. Although discovered in the human genome at single-stranded 3' overhangs in telomeres and in the promoter regions of oncogenes, recent large scale DNA sequencing based studies on human genome have revealed over 700,000 G4 sites\cite{Granotier2005,Siddiqui-Jain2002,Cogoi2006,Sun2011,Chambers2015}. Approximately 10,000 of these sites were captured by the ChIP-seq analysis using a G4 structure-specific antibody; these sites were enriched at regulatory, nucleosome-depleted regions including promoters and 5'UTRs, and in somatic copy number amplified regions\cite{Hansel-Hertsch2016a}. Stabilization of G4 clusters by a G4 ligand followed by whole genome amplification and sequencing also identified 9,651 G4 clusters in the human genome with 3,766 G4 clusters containing at least one transcriptional start site\cite{Yoshida2018}. 

Despite their role in the regulation of cellular processes, G4 DNA can interfere with these processes resulting in replication stalling and transcriptional dysregulation. Several factors are involved, in a cellular process-specific manner, in the resolution of these stable structures to allow for duplex refolding, replication or transcription. A number of \textit{in vitro} and \textit{in vivo} studies have demonstrated the roles of SF1 helicases, PIF1 and DNA2, and SF2 helicases, including RECQ family (BLM and WRN) and Fe-S family (BRIP1 or FANCJ and RTEL1), in enzymatically unwinding G4 DNA structures during replication\cite{Sarkies2012,Piazza2010a,Paeschke2011a,Paeschke2013,Jimeno2018,Mendoza2016a}. G4 unfolding by BRIP1 depends on either REV1 polymerase-mediated G4 destabilization or WRN/BLM helicases\cite{Sarkies2012,Eddy2014}. In the absence of the factors involved in G4 resolution, genome integrity is compromised resulting in chromosomal deletions, rearrangements and mutations. Mammalian cells lacking homologous recombination (HR) factors, BRCA1 and BRCA2, also demonstrate instability at G4 templates in the presence of G4 ligands such as pyridostatin, CX-3543, and CX-5461\cite{Zimmer2015,Xu2017}. Additionally, G4 stabilizers activate ATM- and ATR-mediated DNA damage signaling as well as non-homologous end-joining (NHEJ) pathway to repair the damage\cite{Negi2015,Xu2017}. 

Although DNA G4 structures were proposed as potential drug targets in 1989\cite{Hurley1989a}, G4 stabilizing molecules or G4 ligands became available for clinical use only within the last 15 years. Quarfloxin or CX-3543 was the first G4 ligand to enter phase I/II clinical trial for solid tumors\cite{Drygin2009}. CX-5461 is an orally bioavailable derivative of quarfloxin. It stabilizes G4 structures and inhibits ribosomal RNA (rRNA) synthesis, and has completed a phase I clinical trial for advanced myelomas, non-Hodgkin lymphomas and acute leukemias (ACTRN12613001061729)\cite{Drygin2011,Haddach2012,Bywater2012,Khot2017,Xu2017}. CX-5461 is currently in phase I/II clinical trial (NCT02719977) for BRCA1/2-mutated breast cancers\cite{Xu2017}. In addition, CX-5461 has also shown a promising antitumor effect in preclinical studies against many cancer types, including MYCN-amplified neuroblastomas, ovarian and prostate cancers\cite{Negi2015a,Lee2017,Hein2017,Hald2018,Cornelison2017,Rebello2016}. 

To increase the spectrum of tumors responsive to G4-stabilizers and to overcome resistance associated with targeted therapies, CX-5461 may need to be used in combination with other anti-cancer drugs. Systematic analyses are needed to identify potential targets for combination therapy with CX-5461. While several studies have revealed the cellular roles and consequences of G4 sequences, a comprehensive understanding of how G4 structures are resolved and G4-associated DNA damage is repaired is also lacking. DNA damage and repair factors are frequently altered in cancers. Cancer-associated alterations in factors not yet known to be involved in G4 resolution might also be exploited for precision therapy with G4 stabilizing drugs. 

Pooled genetic screening by RNA interference or gene knockouts is frequently used to identify cancer dependencies, synthetic lethal interactions and drug targets, and to uncover drug resistance mechanisms. Here we performed a subgenomic CRISPR-Cas9 loss-of-function screen to determine sensitivity to two structurally distinct G4 stabilizers, CX-5461 and PDS, in human colorectal cancer cell line, HCT116. Since genome maintenance processes are the most relevant for G4 resolution, we developed a small pooled library with sgRNAs targeting 480 genes involved in chromosome maintenance and genome stability. Parallel screens were performed with BMH-21 which is an inhibitor of RNA polymerase I (POL I) complex and rRNA synthesis without activation of DNA damage stress\cite{Colis2014,Peltonen2014}. Our screen identified genes known for G4 structure resolution, such as REV1, BRCA2, ATM, and ATR. We also discovered that the loss of factors involved in DDR-related ubiquitin signaling, UBE2N (also known as UBC13) and RNF168, sensitizes cells to G4 stabilizers. We further demonstrated that ubiquitination pathway is activated on cellular treatment with G4 stabilizers and that CX-5461, in combination with a UBE2N inhibitor, increases cancer cell toxicity. To date, this is the first gene drop-out CRISPR-Cas9 screen to examine sensitizers to G4 stabilization. Our study has identified the role of non-proteolytic chromatin ubiquitination in G4-DNA resolution and uncovered additional cancer-associated genetic vulnerabilities that could be clinically exploited for treatment with G4-stabilizing drugs.