%%\documentclass[main]{subfiles}

\section*{}
%% Put methods in here.  If you are going to subsection it, use
%% \verb|\subsection| commands.  Methods section should be less than
%% 800 words and if it is less than 200 words, it can be incorporated
%% into the main text.

\subsection{Plasmids.}
The lentiCRISPRv2 was a gift from Feng Zhang (Addgene plasmid \#52961)\cite{Sanjana2014}. A fluorophore-containing variant of lentiCRISPRv2 was generated by replacing the puromycin resistance marker with the DsRed-Express2 (DRX2) fluorescence marker (LCV2-DRX2)
%\cite{Strack2008}
. Gel extraction with the QIAEXII Gel Extraction Kit (Qiagen) was performed on the linearized ~13 kb lentiCRISPRv2 vector following a BamHI and PmeI restriction digest to remove the puromycin marker, along with downstream WPRE and UTR elements. The DRX2 insert was prepared by PCR-amplifying a ~1.6 kb oligo fragment (gBlock, IDT) which included DRX2 sequence as well as the lost WPRE and UTR sequences. Joining of the DRX2 insert and the cut vector was performed using NEBuilder HiFi DNA Assembly (NEB), as per manufacturer's instructions. After transforming the cloning reaction into Stellar chemically competent cells (Clontech) and plating on Amp resistant LB Agar plates, the final clone was selected via a 4-step screening and validation process involving colony PCR, SacII-BamHI restriction digest, Sanger sequencing, and confirmation of DRX2 expression via transient transfection into HEK293T cells.

\subsection{Cell lines and cell culture.}
HCT116 and MDA-MB-231 cell lines were ordered from ATCC.
%MCF7 and HCC1428
UBE2N-deficient HCT116 cells were a kind gift from Dr. Chunaram Choudhary (Novo Nordisk Foundation Center for Protein Research, University of Copenhagen). Dox-inducible GFP-K63-Super-UIM U2OS cells were a kind gift from Dr. Niels Mailand (Novo Nordisk Foundation Center for Protein Research, University of Copenhagen). HCT116 and U2OS cells were cultured in McCoy's media supplemented with 10\% fetal bovine serum (FBS), HEK293T and MDA-MB-231 cells in DMEM containing 10\% FBS. 
%HCC1428 cells in RPMI with 10\% FBS, MCF7 cells in EMEM containing 0.01 mg/ml insulin and 10\% FBS
All cells were cultured at \SI{37}{\celsius} with 5\% CO2. All cell lines were tested for mycoplasma.

\subsection{Drugs.}
CX-5461 (Selleckchem, cat\# S2684) and BMH-21 (eMolecules) were prepared in \SI{50}{\milli\Molar} \ch{NaH2PO4}, as described previously\cite{Xu2017}.  Pyridostatin trifluoroacetate (Sigma-Aldrich, cat\# SML0678) was prepared in water, NSC697923 (Sigma-Aldrich, cat\# SML0618) in DMSO, and puromycin (ThermoFisher, cat\# A1113803) in water. 

\subsection{Sub-genomic library sgRNA oligonucleotide design and synthesis.}
20mer oligonucleotide sequences representing sgRNAs were selected from Human GeCKO v2 pooled library\cite{Sanjana2014}. sgRNA designs missing from the GeCKO library for DNA2, TERF2, DBF4, and TRRAP genes were added\cite{Wang2014}. DNA oligonucleotides were synthesized as a pool using a B3 synthesizer (CustomArrayInc) platform, PCR amplified using ArrayF and ArrayR primers and Phusion HS Flex (NEB) and size selected using 1\% agarose gel as described previously\cite{Shalem2014,Sanjana2014}. Pooled oligos were cloned into lentiCRISPR v2-puromycin plasmid, digested with \textit{BsmbI}, at 42:1 insert-to-vector ratio, using Gibson assembly\cite{Shalem2014}. Gibson reaction was diluted 4-fold to transform \SI{25}{\micro\litre} of Lucigen Endura electrocompetent cells. \SI{1}{\milli\litre} of transformation was grown in \SI{100}{\milli\litre} of LB-ampicillin at \SI{37}{\celsius} overnight and plasmid DNA was extracted using Endofree Plasmid Maxi Kit (Qiagen).  

\subsection{Library lentivirus generation and titration.}
HEK293T cells were transfected, at 80\% confluency, with \SI{20}{\micro\gram} sgRNA plasmid pool, \SI{10}{\micro\gram} pMD2.G-VSVG and \SI{15}{\micro\gram} psPAX2 using TransIT-LT1 reagent and reduced serum OptiMEM media in ten \SI{15}{\centi\meter\squared} tissue culture dishes. 24 hours later, media was replaced with collection media (DMEM, 10\% FBS and 1\% BSA). Media containing virus was collected 48 and 72 hours later, pooled and filtered through \SI{0.45}{\micro\meter} PVDF membrane. Virus was then concentrated 240x by ultracentfugation at 24,000 RPM for 90 minutes and resuspension in virus collection media for 30 minutes on shaker at room temperature. Aliquots were stored at \SI{-80}{\celsius}. 
In order to determine virus titers to transduce HCT116 cells at an MOI of 0.3, 1 million cells were plated per well into a 12-well plate. 5-fold dilutions of the virus were added to each well (\SI{0.00064}{\micro\liter} to \SI{50}{\micro\liter}) along with a no transduction control. After 24 hours of incubation with the virus, cells in each well were split 1:10 into duplicate wells, one with \SI{0.5}{\micro\gram} puromycin and other with no puromycin. After three days, cell counts were determined. Survival fraction was determined by dividing the cell count from the replicate with puromycin by cell counts from the replicate without puromycin. Viral titer was determined using (survival fraction x number of cells per well x 1000/volume of virus per well).  

\subsection{Dropout screens.}
For dropout screens, 30 million HCT116 cells were plated at a density of 1 million cells per well in 12-well plates and transduced with \SI{30}{\micro\liter} LCV2-puro library at MOI of 0.3. After 24 hours, cells were pooled and plated in 150 cm2 plates with fresh media containing \SI{0.5}{\micro\gram}/ml puromycin. Cells were pooled after 3 or 6 days of puromycin selection and cultured until day 19 in triplicates. Cells were passaged at regular intervals on days 4, 7, 10, 13 and 16. At each passage, 2 to 3 million cells were plated to maintain a representation of $\sim$ 700 to 1000 cells per sgRNA. Cells harvested at each passage and on day 19 were used to make DNA for sequencing library preparation.
For drug screens, 50 million cells were transduced with \SI{50}{\micro\liter} of the virus as described above followed by 3-days of puromycin selection. On day 5 after transduction, drug/vehicle treatments were started in triplicates and continued until day 19. Fresh growth media with drugs was added at the time of splitting cells on days 12 and 15 post-transduction. Drugs were added at following concentrations: vehicle \ch{NaH2PO4}, CX-5461 \SI{23}{\nano\Molar} (IC$_{30}$), CX-5461 \SI{33}{\nano\Molar} (IC$_{50}$), PDS \SI{7.73}{\micro\Molar} (IC$_{30}$), PDS \SI{11.92}{\micro\Molar} (IC$_{50}$) and BMH-21 \SI{59}{\nano\Molar} (IC$_{30}$). In parallel, WT and BRCA2-null HCT116 cell lines were treated with similar doses to determine sensitivity. Cells were harvested on days 1, 4, 5, 12, 15 and 19 to make DNA for downstream sequencing library preparation. 

\subsection{Sequencing library preparation and next generation sequencing.}
\SI{1}{\nano\gram} of pooled sgRNA plasmid library was PCR amplified using a multiplex PCR with \SI{0.3}{\micro\Molar} mix of nine forward primers with stagger (v2NGS-F1 to v2NGS-F9) and one reverse primer v2NGS-R10, and 1x Platinum Multiplex PCR master mix (Life Technologies, cat\#4464269) in a total of \SI{100}{\micro\liter} reaction. The PCR conditions were: \SI{95}{\celsius} 2 min; 20 cycles of \SI{95}{\celsius} 30 seconds, \SI{60}{\celsius} 90 seconds, \SI{72}{\celsius} 60 seconds; \SI{72}{\celsius} for 10 minutes. PCR products were checked on agarose gel for the presence of the 218 bp band. \SI{5}{\micro\litre} of PCR products were cleaned using Exosap-IT (ThermoFisher Scientific, cat\#78201.1.ML). Purified PCR products were barcoded using Illumina Nextera barcodes and Roche FastStart HiFi PCR reagents (cat\#04738292001). Barcoded PCR products were purified on E-gel. Library quality analysis was done using Agilent Bioanalyzer. The library was sequenced on MiSeq. 
For dropout screens, genomic DNA was extracted from frozen cell pellets using Blood and Cell Culture DNA kit 500G (Qiagen). \SI{10}{\micro\gram} of genomic DNA (representing approximately 455x coverage per sgRNA) for each sample was PCR amplified and barcoded as described above. Barcoded PCR products were purified on E-gel or on SPRIselect beads (Beckman Coulter). Pilot dropout libraries were screened on MiSeq. Drug screen libraries were sequenced on NextSeq using NextSeq High Output 75v2, 75 cycles, single-end with 40\% of phiX. 

\subsection{Screen data processing and analysis.}
Unique sgRNA sequences were extracted from raw FASTQ files and mapped to the reference library FASTA index using CaRpools v0.83 and Bowtie2 v2.2.9 with parameters to not allow any mismatch\cite{Winter2016,Langmead2012a}. The sgRNA read count files generated by CaRpools were used to perform hit analysis and generate data statistics using MaGeCK v0.5.4 with default parameters\cite{Li2014}. 
For downstream analysis including gene set enrichment and overlap between GQ-drugs, candidate genes from each treatment group (CX-5461 IC$_{50}$ and IC$_{30}$, PDS IC$_{50}$ and IC$_{30}$, and BMH-21 IC$_{30}$) with a P-value of $<$0.045 in the MaGeCK gene rankings were combined into an 81-gene list.  

\subsection{Gene set enrichment analysis.}
Gene Ontology (GO biological process 2015, GO cellular component 2015, GO Molecular Function 2015) and KEGG pathways (2016) enrichment analysis for top hits from the drug screens was performed using Enrichr (http://amp.pharm.mssm.edu/Enrichr/)\cite{Chen2013,Kuleshov2016}. To identify pathways and ontologies specific to G-quadruplex stabilization, gene hits that were unique to and in common with the BMH-21 treatment group (23 genes) were excluded from enrichment analysis. For the remaining set of 58 genes, each gene was assigned a weight/membership value based on the number of G4-stabilizing treatment groups that gene was represented in. Genes represented in four, three, two and one treatment group(s) were given values of 1, 0.75, 0.5 and 0.25, respectively. The P-values were computed from Fisher exact test and adjusted using the Benjamini-Hochberg method for correction for multiple hypotheses testing. The top enriched categories were ranked by the combined score which was calculated using the P-value and z-score of the deviation from the expected rank (ln(p) * z). To identify pathways and ontologies specific to BMH-21, 19 gene hits that were unique to the BMH-21 treatment group were used for enrichment analysis as described above.

\subsection{Subcloning, lentivirus production and titration for individual sgRNAs.}
Individual sgRNAs were designed using the Deskgen Cloud CRISPR Design Software (Desktop Genetics). sgRNA cloning into LCV2-DRX2 vector was performed as previously described\cite{Sanjana2014,Shalem2014}. Transformation was performed by adding \SI{2}{\micro\litre} ligated product into \SI{50}{\micro\litre} Stellar Chemically Competent Cells (Takara) as per manufacturer's instructions. Up to \SI{100}{\micro\litre} of transformed cells in S.O.C. Medium (ThermoFisher) were plated onto LB Agar plates containing \SI{100}{\milli\gram}/ml Ampicillin and grown overnight at \SI{37}{\degreeCelsius}. In the following afternoon, multiple colonies from each plate of cloned sgRNA plasmids were inoculated and grown overnight at \SI{37}{\degreeCelsius} in \SI{1}{\milli\litre} LB broth with \SI{100}{\milli\gram}/ml Ampicillin. Plasmid DNA was extracted from \SI{800}{\micro\litre} bacteria culture using Qiagen Plasmid Miniprep Kit (Qiagen) or Monarch Plasmid Miniprep Kit (NEB), as per manufacturer's instructions. Plasmid samples were sent for Sanger sequencing validation, and the correct clones were expanded, by growing the remaining \SI{200}{\micro\litre} bacteria culture in \SI{200}{\milli\litre} LB Broth overnight, at \SI{37}{\degreeCelsius} with shaking. Plasmid DNAs were then extracted using the PureLink HiPure Plasmid Maxiprep Kit (ThermoFisher). Lentivirus was generated as described above. To titrate lentiviruses, cells of interest were plated at 50,000-200,000 cells per well into a 24-well plate. Starting from \SI{10}{\micro\litre}, 5-fold dilutions of the virus were added into \SI{250}{\micro\litre} of media (up to 8 dilutions). After 24 hours, media were replaced with fresh media without virus. Seventy-two hours after viral transduction, flow cytometry analyses were performed using FlowJo to determine the percentage of cells expressing DRX2. Virus titration curves were generated by plotting the fluorophore-expressing fraction against the dilution factor. From the data points on the exponential part of the curve, viral titer was calculated by using (\% positive cells x \# cells per well x 1000 / volume of virus per well).

\subsection{Competitive growth assays (CGA).}
LCV2-DRX2 vectors with individual sgRNAs were produced as described above. HCT116 cells were transduced in 24-well plates at a density of 150,000 cells per well with the virus at an MOI of 0.1-0.3. Three days after transduction, cells expressing DRX2 were flow sorted on Aria Fusion or Aria III and mixed with non-transduced cells. First, the ratio of transduced to non-transduced cells was determined to allow the untreated mixtures to contain at least 40-50\% transduced cells at the end of the assay by monitoring the proportion of DRX2+ cells 4 and 7 days after transduction using Fortessa flow cytometer. The sgRNA MMS22L.2 was excluded from the final analysis because of the low proportion of DRX2+ cells in the mixed cell population.  Growth-adjusted cell mixtures were split and plated in triplicate wells to be treated with drugs or vehicle as in the drug screens. Cells were harvested 7 days after drug treatment and analyzed by flow analysis to determine the fraction of cells expressing DRX2. The ratio of the mean percentage of DRX2+ cells in drug-treated and vehicle-treated cell populations was calculated to determine the relative fold-change in the mixed cell populations for each sgRNA of interest. Results were analyzed using R. Each experiment was repeated at least twice with three technical replicates per experiment.   

\subsection{Statistical analysis for CGA.}
Bootstrapping was performed in a blocked fashion, to reflect the conditions of the experiment. For an experiment involving NA (Not Available) assays, NA assay numbers were randomly selected with replacement (outer bootstrap block). Within the NA randomly selected assays, the NR replicate values were then randomly selected with replacement (inner bootstrap block). The resultant data set was of the same form as the original, with the same number of assays and replicate observations within each assay. 400,000 blocked bootstrap replicate data sets were constructed and the average proportion of cells surviving calculated in the logistic space. After transforming the averages back to the raw proportion space, differences between vehicle and drug conditions were calculated. From the distribution of the 400,000 bootstrapped differences, confidence intervals were determined. Confidence intervals of 95\%, 99\%, 99.9\% and 99.99\% (corresponding to type I error
rates 0.05, 0.01, 0.001, 0.0001 respectively) were determined. With 5 drug conditions to compare to the vehicle, across 10 sgRNA conditions, 50 results are being assessed. A Bonferroni adjustment for 50 comparisons means the greatest difference must show a significance level of 0.05/50 = 0.001 for a false discovery rate (FDR) of 5\% or 0.01/50 = 0.0002 for FDR = 1\%.

\subsection{Drug dose response analysis.}
Cells were plated at low densities in 96-well plates (HCT116 cells at 300 cells/well, MDA-MB-231 at 300/well) in three or four replicates.
%MCF7 at 2,000/well, HCC1428 at /well
24 hours later  serial dilutions of drugs  were added in three or four replicates per treatment. The drug media was refreshed every 3 or 4 days. At the indicated time points, cells were assayed with WST-1 reagent (Roche, cat\#CELLPRO-RO) on the plate reader. The results were analyzed using GraphPad Prism. The means for drug-treated replicates were divided by that of the vehicle-treated populations for each genotype. IC50 values were determined by fitting a four-parameter logistic curve (variable slope model) to the data.   

\subsection{Immunoblotting.}
For chromatin ubiquitination experiments, cells were resuspended in Triton Extraction Buffer (TEB) at a concentration of 10\textsuperscript{7} cells/mL and lysed on ice for 10 minutes on a horizontal shaker at 90 rpm. Cells were then washed in TEB and resuspended in 0.2N HCl at 4x10\textsuperscript{7} cells/mL before extracting histones overnight at \SI{4}{\celsius}. After centrifugation, the supernatant was saved and neutralized with 1/10th the volume of 2M NaOH. For each sample, 10\textsuperscript{6} cells were boiled in 1X Laemmli buffer and 2-mercaptoethanol at a 1:10 ratio for 10 minutes at \SI{90}{\celsius} before loading in each well of SDS-PAGE. For all other Western blots, cells were boiled in 1X Laemmli buffer for 10 minutes at \SI{90}{\celsius} and protein was quantified using the Pierce 660 nm Protein Assay (ThermoFisher, cat\#22660). \SI{10}{\micro\gram} protein was loaded in each well of a 12\% SDS-PAGE and transferred to nitrocellulose or PDMS membrane at 100V for 1.5 hours or at 30V overnight. Primary antibodies were incubated at the concentrations listed in Table ~\ref{table:Western_antibodies} as per manufacturer's instructions. After incubating with the appropriate secondary antibodies (Goat anti-mouse, Dako cat\#P0447 and Abcam cat\#ab97040; goat anti-rabbit, Abcam cat\#ab6721; mouse anti-goat, Santa Cruz Biotechnology cat\#sc-2354) at 1:10000 or 1:5000 in 5\% non-fat dried milk-TBST for 1 hour, membranes were visualized with Immobilon Western Chemiluminescent HRP Substrate (MilliporeSigma, cat\#WBKL20500) and imaged with the ImageQuant LAS 4000 (GE Healthcare) using the ImageQuant TL software. 
%Following antibodies were used: GAPDH (Santa Cruz, sc-48166 at 1:1000 and Millipore Sigma, MAB374 at 1:5,000), H2A (Abcam, ab18255 at 1:1,000), H2AX (Bethyl, A300-082A at 1:6,000), H2AUb-K15 (Millipore Sigma, MABE1119 at 1:500), H2AUb-K119 (Millipore Sigma, 05-678 at 1:750), gamma-H2AX (Abcam ab81299 at 1:10,000), H2B (Abcam, ab1790 at 1:30,000), H3 (Abcam, ab1791 at 1:30,000), RNF168 (Abcam, ab58063 at 1:1000 and Abcam, ab220324 at 1:500), RNF8 (Millipore Sigma, 09-813 at 1:3,000), UBE2N (ThermoFisher Scientific, 37-1100 at 1:2,000), FK2 (Enzo Life Sciences, BML-PW8810-0100 at 1:500), Ubiquitin, linkage-specific K63 (Abcam, ab179434 at 1:1,000), Vinculin (Sigma, V9131 at 1:10,000)

\subsection{Statistical Analysis for FK2 Immunoblots.}
Increasing levels of H3 were associated with decreasing levels of FK2 peak density measurements (FK2 peak at 25kDa), so H3 did not appear appropriate as a loading control. Total H2A measured via 2 bands at 15kDa and 2 bands measured at 30kDa (representing modified H2A) provided a loading control positively associated with FK2 peak. With two data points per experimental condition, independent linear fits within condition were not possible as all fits would have been degenerate, leaving no degrees of freedom for error estimation.  Thus the simplifying assumption of equal slopes within all treatment conditions was made. Initial analysis on the raw western blot photo density data yielded estimates with confidence intervals (CI) extending to negative density values. Transforming the photo density values via the base 2 logarithm transformation removed the lower bound of zero, yielding an unconstrained data range from minus to plus infinity. Regression model fit diagnostics showed reasonable Gaussian behaviour for the log2 transformed data, with estimates and CI well removed from data boundaries.  Treatment condition differences in the log realm correspond to fold-change estimates back in the raw scale, facilitating interpretation of differences as compared to differences expressed in arbitrary densitometry values. With limited data available, basic linear model fits were obtained. Insufficient data was available to perform e.g. linear mixed effects models.  All conditions were estimated as fixed effects in an ANCOVA model (FK2 peak ~ Total.H2A + Genotype * Treatment) with total H2A providing the ANCOVA covariate to allow for loading control adjustment. The model fit to the log2 transformed data yields the following ANOVA table:
\newline
Analysis of Variance Table
\newline
Response: log2FK2 peak
\begin{center}
\begin{tabular}{ c c c c c c c}
 & Df & Sum Sq & Mean Sq & F-value & Pr($>$F) \\
log2H2A & 1 & 3.8215 & 3.8215 & 8.2261 & 0.01319 & * \\
Genotype & 1 & 2.0349 & 2.0349 & 4.3803 & 0.05653 & . \\
Trt & 6 & 1.5716 & 0.2619 & 0.5638 & 0.75196 & .. \\ 
Genotype:Trt & 6 & 1.2510 & 0.2085 & 0.4488 & 0.83337 & .. \\
Residuals & 13 & 6.0393 & 0.4646 & .. \\
\end{tabular}
\end{center}
Significance codes:  0 *** 0.001 ** 0.01 * 0.05 . 0.1 .. 1
\newline
The positive association of log2(FK2 peak) with log2(Total H2A) shows a p-value of 0.013, indicating that the ANCOVA adjustment was important and necessary. The linear fits shown in the Figure~\ref{fig:chromatin-ubiquitination}b show that ubiquitination levels in the CRISPR knockout UBE2N-/- line were generally lower than those in the wild type line, and this is reflected in the Genotype p-value 0.057 in the ANOVA table. The differences associated with differing treatment conditions show wide CI, all crossing the null hypothesis value for no treatment difference (Fold-change = 1.0 on the raw scale, or zero difference in intercept values on the log2 scale).  Thus there is no statistical evidence of a difference in ubiquitination levels due to treatment conditions relative to the wild type vehicle control condition.  

\subsection{Immunofluorescence.}
For RNF168 foci, U2OS cells were treated with CSK buffer (100mM NaCl, 300mM sucrose, 3mM MgCl, 10mM PIPES (pH 6.8), with proteinase inhibitors) for 4 minutes followed by fixation with 2\% paraformaldehyde in TBS (50mM Tris-HCl, pH7.5, 150mM NaCl) for 20 minutes, and methanol for 1 minute. After blocking in 3\% BSA and 0.2\% Tween-20 in TBS, RNF168 antibody (Table~\ref{table:Western_antibodies}) was incubated overnight followed by incubation with secondary antibody for 1hour. Immunofluorescence of BG4, 53BP1 and $\gamma$-H2AX was described previously\cite{Xu2017}. To visualize FK2 foci, U2OS cells grown on glass cover slides were fixed with cold methanol for 10 minutes and then with acetone for 30 seconds. Cells were permeabilized with 0.5\% Triton X-100 for 25 minutes, then blocked for 1 hour with 3\% BSA and 0.2\% Tween-20 in TBS. FK2 antibody (Table~\ref{table:Western_antibodies}) was incubated at 1:10,000 dilution overnight. Mouse Alex488 secondary antibody was diluted at 1:5000 dilution and incubated for 1 hour. Because of background foci, damage foci positive cells were defined as $\gamma$-H2AX foci $\geq$ 5, 53BP1 foci $\geq$ 3, RNF168 foci $\geq$ 3, FK2 foci $\geq$ 10. At least 100 cells were counted for each condition, and at least two independent experiments were performed. P-value was determined by one-tailed unpaired t-test for the difference of means. For FK2 foci analysis in case of CX-5461 \SI{1}{\micro\Molar}, Welch's one-tailed test was used to determine significance because of unequal variances.  
For K63-linked ubiquitination, 30,000 U2OS cells with GFP-K63-Super-UIM were grown on coverslips and induced by doxycycline as described previously\cite{Thorslund2015}. Drugs or vehicle were added 24 hours after induction and cells were fixed in 4\% paraformaldehyde for 15 minutes    followed by permeabilization with 0.2\% Triton X-100. Coverslips were mounted on glass slides with DAPI and images were obtained using an upright Colibri LED microscope. All images were taken using the same exposure time. Cell quantification was performed using ImageJ. For each condition, at least 100 cells were examined with a threshold of 10 or more foci per cell. Two independent experiments were performed. P-value was determined by one-tailed unpaired t-test for the difference of means.

\subsection{Drug synergy.}
HCT116 cells were plated in 96 well plates at a density of 800 cells per well. Next day, serial dilutions of NSC697923 and CX-5461 were prepared and drug mixtures in 45 combinations were added to cells. NSC697923 was added at five doses: IC$_{50}$ (166.1 nM, IC$_{50}$x2 (332.2 nM), IC$_{50}$x4 (664.4 nM), IC$_{50}$x0.5 (83.05 nM) and IC$_{50}$x0.25 (41.53 nM).CX-5461 was added at nine doses: IC$_{50}$ (7.4 nM), IC$_{50}$x2 (14.8 nM), IC$_{50}$x4 (29.6 nM), IC$_{50}$x8 (59.2 nM), IC$_{50}$x0.5 (3.7 nM), IC$_{50}$x0.25 (1.85 nM), IC$_{50}$x0.125 (0.93 nM), IC$_{50}$x0.0625 (0.46 nM) and IC$_{50}$x0.03125 (0.23 nM). All individual drug doses and no drugs controls were added in parallel. After four days of drug treatment, WST-1 assay was performed and results were analyzed using MacSynergyII software.   

%% Here is a description of a specific method used.  Note that the
%% subsection heading ends with a full stop (period) and that the
%% command is \verb|\subsection{}| not \verb|\subsection{}|.